build.gradle

```diff
@@ -44,6 +44,7 @@ ext {
 
     junitVersion = "4.12"
     mockitoVersion = "2.26.0"
+    //byteBuddyVersion = "1.9.10" // transitive dependency of mockito-code 2.26.0
```

Comment:
Remove commented line?

Reply:
yep. was just waiting for the CI to cooperate before I remove this